rpy/robjects/pandas2ri.py

```diff
@@ -141,13 +142,15 @@ def py2rpy_pandasseries(obj):
         }[homogeneous_type](obj)
     elif obj.dtype.name in integer_array_types:
         if not obj.dtype.numpy_dtype.isnative:
-            raise(ValueError('Cannot pass numpy arrays with non-native byte orders at the moment.'))
+            raise(ValueError('Cannot pass numpy arrays with non-native byte'
```

Comment:
May be something like the following would make it work instead ?

```
obj = copy(obj)
obj.values.byteswap(inplace=True)
```

Reply:
Actually shouldn't that conversion belong to the numpy converter ?

Reply:
not sure. I copied this from your numpy code: https://github.com/rpy2/rpy2/blob/master/rpy/robjects/numpy2ri.py#L57

Reply:
I guess I should add the call to `byteswap` to the numpy converter then.

Where I might be getting confused is whether this is pandas-specific step here. If not, we have numpy arrays of integers and the conversion should be delegated to the numpy converter (without reimplementing it).

Reply:
It looks like I misunderstand something about `dtype.isnative` and/or `byteswap()`: calling `byteswapping` does not invert the boolean `dtype.isnative`.

Reply:
My comment should just be ignored then. :)